verification/curator-service/ui/src/components/App/App.tsx

```diff
@@ -619,7 +626,11 @@ export default function App(): JSX.Element {
                             <a
                                 className={classes.mapLink}
                                 data-testid="mapLink"
-                                href="https://map.covid-19.global.health/"
+                                href={
+                                    env === 'prod'
+                                        ? 'https://map.covid-19.global.health/'
+                                        : 'http://dev-map.covid-19.global.health/'
```

Comment:
We will have a QA env soon too so perhaps this should be an object where the keys are the env names and the values are the URLs